### DISCUSSION TITLE
[2023-08-29] GUI Team Demo Day

### DISCUSSION THREAD

**Discussion by vitvakatu** [enso-org/enso#7687] [Weekly Demo Day]
@MichaelMauderer:

- https://github.com/enso-org/enso/pull/7619
- https://github.com/enso-org/enso/pull/7620
- https://github.com/enso-org/enso/pull/7644

![](https://user-images.githubusercontent.com/1428930/262675635-d0f19205-85f3-4046-b3ef-7540030c9ab5.gif)

![](https://user-images.githubusercontent.com/1428930/262025991-a1e5aa7e-8776-4438-9c1e-80ac1afd0292.gif)

Me:

- https://github.com/enso-org/enso/pull/7618. Fixed a few issues spotted on review and also added support for pasting nodes as plain text to other applications.
- https://github.com/enso-org/enso/pull/7670
- https://github.com/enso-org/enso/pull/7680

https://github.com/enso-org/enso/assets/6566674/4e7addf9-2175-4f2a-a571-4ef823de5cb0

https://github.com/enso-org/enso/assets/6566674/273abfaa-45b6-4374-8d4c-3b8c4e2c1fc2

**Comment by kazcw** [comment]
- Context restoration (#7255):

[vokoscreenNG-2023-08-25_09-39-11.webm](https://github.com/enso-org/enso/assets/1047859/cfa90ec5-72a1-41e6-bafa-177fa5e85fb2)

- Currently debugging a rendering issue (#7642). SpectorJS is not working correctly, but I think I have the solution.

**Comment by Frizi** [comment]
Preparing Enso IDE prototype written in Vue:

<img width="590" alt="image" src="https://github.com/enso-org/enso/assets/919491/62af7ea3-627e-4bad-8e90-2733b685ddf0">

**Comment by hubertp** [comment]
Backend:
- Displaying only static methods when creating new node without self-argument #7632 
- `opt1._` crash #7582 (seems to be triggered by IDE)
